deploy/kubernetes/manifests/loadtest-dep.yaml

```diff
@@ -13,6 +13,9 @@ metadata:
   namespace: loadtest
 spec:
   replicas: 2
+  selector:
+    matchLabels:
+      name: loadtest  
```

Comment:
Also there should be "load-test", because of error "Invalid value: map[string]string{"name":"load    -test"}: `selector` does not match template `labels`"